### PR TITLE
MRG fix order of plot_ica_properties for <py36

### LINKED ISSUE #5401: ica.plot_properties only plots 3 epochs with raw input?
Try:

```
import mne
path_eeglab = mne.datasets.testing.data_path()

fname = path_eeglab + '/EEGLAB/test_raw.set'
montage = path_eeglab + "/EEGLAB/test_chans.locs"

event_id = {"rt": 1, "square": 2}
raw = mne.io.read_raw_eeglab(fname, event_id=event_id, preload=True, montage=montage)
ica = mne.preprocessing.ICA(20, random_state=0).fit(raw.filter(1, None))
ica.plot_properties(raw, 0)
```

Gives:

unknown-191

I.e., for some reason, only 3 "epochs" are plotted.

@mmagnuski

### PR DESCRIPTION
**Pull request by jona-sassenhagen**
Closes #5401

@mmagnuski

### CODE REVIEW

File: mne/viz/ica.py [hunk 1]
```diff
@@ -132,7 +133,7 @@ def _plot_ica_properties(pick, ica, inst, psds_mean, freqs, n_trials,
                          set_title_and_labels, plot_std, psd_ylabel,
                          spectrum_std, topomap_args, image_args, fig, axes):
     """Plot ICA properties (helper)."""
-    topo_ax, image_ax, erp_ax, spec_ax, var_ax = axes
+    erp_ax, image_ax, spec_ax, topo_ax, var_ax = axes
```
Comment (mmagnuski) [inline mne/viz/ica.py]: That changes also the order of axes passed by the user. An older order is still in the docs of `axes` arg. Ideally the user would not have to change the order (the previous one had a spatial logic to it, which is more intuitive than alphabetical :) ).

Comment (jona-sassenhagen) [reply mne/viz/ica.py]: Hm. Ok. I'll do it.

**Comment by mmagnuski** [comment]
@jona-sassenhagen You seem to be able to travel back in time (look at the message time):
image

**Comment by jona-sassenhagen** [comment]
8)

File: mne/viz/ica.py [hunk 2]
```diff
@@ -350,7 +350,7 @@ def set_title_and_labels(ax, title, xlab, ylab):
 
         # the actual plot
         fig = _plot_ica_properties(
-            pick, ica, inst, psds_mean, freqs, len(picks),
+            pick, ica, inst, psds_mean, freqs, ica_data.shape[1],
```
Comment (mmagnuski) [inline mne/viz/ica.py]: If that controls the y axis scale shouldn't it be `inst._data.shape[0]` or `_segment_raw(inst, ...).shape[0]` (the number of epochs)?

Comment (jona-sassenhagen) [reply mne/viz/ica.py]: At that time point, ica_data is:

`ica_data = np.swapaxes(epochs_src.get_data()[:, picks, :], 0, 1)`

Comment (mmagnuski) [reply mne/viz/ica.py]: ok!

**Comment by agramfort** [comment]
merge if you're happy

**Comment by jona-sassenhagen** [comment]
Would like to hear if it also fixes the problem for @mmagnuski ?

**Comment by mmagnuski** [comment]
I'll check tomorrow and let you know.

**Comment by jona-sassenhagen** [comment]
Ok, feel free to merge if it does.

**Comment by mmagnuski** [comment]
Works perfectly!

**Pull request merged**